### PR TITLE
task/98 - create module presentation

### PR DESCRIPTION
**Pull request by jordanrafalm**

### CODE REVIEW

File: app/src/main/java/com/safetyheads/presentation/utils/AndroidAcademyApplication.kt [hunk 1]
```diff
@@ -1,11 +1,11 @@
-package com.safetyheads.akademiaandroida
```
Comment (msarelo) [inline app/src/main/java/com/safetyheads/presentation/utils/AndroidAcademyApplication.kt:1]: `AndroidAcademyApplication` should stay in app module.

**Review by msarelo — CHANGES_REQUESTED** [review]
- Test should be also moved
- Main application and navigation also should be in the app module

File: presentation/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application
```
Comment (msarelo) [inline presentation/src/main/AndroidManifest.xml:4]: it should stay in app module, we don't need <application/> in module

File: presentation/src/main/res/drawable-v24/ic_launcher_foreground.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (msarelo) [inline presentation/src/main/res/drawable-v24/ic_launcher_foreground.xml:1]: launcher should be in the app module

File: presentation/src/main/res/values-night/themes.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<resources>
+    <!-- Base application theme. -->
```
Comment (msarelo) [inline presentation/src/main/res/values-night/themes.xml:2]: it also should be in the app module

**Review by msarelo — CHANGES_REQUESTED** [review]

**Review by forceindia712 — DISMISSED** [review]

**Pull request closed without merging**